ecosystem/sep-0010.md

```diff
@@ -97,12 +99,20 @@ On success the endpoint must return `200 OK` HTTP status code and a JSON object
   * source account set to server's signing account
   * invalid sequence number (set to 0) so the transaction cannot be run on the Stellar network
   * time bounds: `{min: now(), max: now() + 900 }` (we recommend expiration of 15 minutes to give user time to sign transaction)
-  * operations: `manage_data(source: client_account, key: '<home domain> auth', value: random_nonce())`
-    * The value of key is the fully qualified domain name of the web service requiring authentication followed by `auth`. It can be at most 64 characters.
-    * The value must be 64 bytes long. It contains a 48 byte cryptographic-quality random string encoded using base64 (for a total of 64 bytes after encoding).
+  * operations:
+    * `manage_data(source: client_account, key: '<home domain> auth', value: random_nonce())`
+      * The value of key is the fully qualified domain name of the web service requiring authentication followed by `auth`. It can be at most 64 characters.
```

Comment:
Note: if we decide to remove the restriction of what goes in the "key" param on SEP v2.1.0, we should also port that change here.